src/invocation/store.rs

```diff
@@ -1,134 +1,7 @@
 //! Storage for [`Invocation`]s.
 
-use super::Invocation;
-use crate::ability;
-use crate::{crypto::varsig, did::Did};
-use libipld_core::{cid::Cid, codec::Codec};
-use std::sync::{Arc, RwLock, RwLockReadGuard, RwLockWriteGuard};
-use std::{collections::BTreeMap, convert::Infallible};
+mod memory;
+mod traits;
 
-pub trait Store<T, DID: Did, V: varsig::Header<C>, C: Codec + Into<u64> + TryFrom<u64>> {
```

Comment:
Just made the modules layout the same as all of the other stores.

Reply:
Haha, same. I'd be working in `delegation/store/memory.rs` and search forever for the invocation store (because I would think it'd be in `invocation/store/memory.rs`, so I'd fuzzy-search "invocationmemory", but... :sweat_smile:)